packages/invoices/src/InboundShipment/api/hooks.ts

```diff
@@ -191,7 +196,7 @@ export const useInbounds = () => {
   const api = useInboundApi();
 
   return {
-    ...useQuery(['invoice', 'list', api.storeId, queryParams], () =>
+    ...useQuery(api.keys.list(queryParams), () =>
```

Comment:
use `queryParams` actually seems not ideal.. it's a massive object with a bunch of duplicate things which gets diff'd all the time..

I think it would be a better idea to try and align this better with the generated gql code - i.e. `PaginationInput` etc..

I had a quick look but it'd take a bit more work to untangle all the filtering/sorting/pagination so have left that - likely something to tackle with #936 , I think

Reply:
Something that would be good to look at with #936 I agree. 
The `queryParams` is ok for now but could be simplified a bit for use in the key.

Was wondering about alignment with the gql types and if that helps anything

Reply:
I wouldn't mind a query params 'system' that would allow for better typing of sort keys etc which would likely require using the gql types